akka-actor/src/main/java/akka/dispatch/forkjoin/ForkJoinWorkerThread.java

```diff
@@ -7,115 +7,104 @@
 package akka.dispatch.forkjoin;
```

Comment:
is it possible to disable formatting of certain files, sections?

Would be good to keep this (forkjoin/*) as close as the original copy as possible, to make diff easier.

Reply:
So does the protobuf one i think.